cmd/log-parser/hexbytes.go

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) 2017-2018 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"errors"
+	"io"
+	"io/ioutil"
+	"os"
+	"strings"
+)
+
+// HexByteReader is an I/O Reader type.
+type HexByteReader struct {
```

Comment:
just curious, this is not something we could use package "encoding/hex" for is it?
(which I've used, but had some fun with before, as it only handles a single case (lower iirc), and I needed it to handle upper :-) )

Reply:
I don't think that package handles escaped hex values?